logsmanagement/stress_test/run_stress_test.sh

```diff
@@ -0,0 +1,166 @@
+#!/bin/bash
+
+# Default configuration options
+DEFAULT_BUILD_CLEAN_NETDATA=0
+DEFAULT_BUILD_FOR_RELEASE=1
+DEFAULT_NUM_LOG_SOURCES=0
+DEFAULT_DELAY_BETWEEN_MSG_WRITE=1000000
+DEFAULT_TOTAL_MSGS_PER_SOURCE=1000000
+DEFAULT_QUERIES_DELAY=3600
+DEFAULT_LOG_ROTATE_AFTER_SEC=3600
+DEFAULT_DELAY_OPEN_TO_WRITE_SEC=6
+DEFAULT_RUN_LOGS_MANAGEMENT_TESTS_ONLY=0
+
+if [ "$1" == "-h" ] || [ "$1" == "--help" ]; then
+  echo "Usage: $(basename $0) [ARGS]..."
```

Comment:
Pleaase, take a look in every `shellcheck` warning

Reply:
Fixed!